### ISSUE TITLE
ActionButton icon does not use initialized icon subset

### ISSUE THREAD

**Issue by AndrewCraswell** [OfficeDev/office-ui-fabric-react#10810]
### Environment Information

- **Package version(s)**: 7.38.0
- **Browser and OS versions**: Chrome, Windows 10

#### Actual behavior:

ActionButton component forces the full Fabric MDL2 icon font woff2 file to download, even when a custom font subset has been initialized using the same icon.

I tested replacing the ActionButton component with a Icon component and passing the same value for `iconName`. Using the Icon component it correctly uses the icon in the initialized subset. It seems the ActionButton is looking for a CSS class with the same name rather than hooking into the reigstered icons. Sites that also use the Fabric stylesheet will load the whole font set in this scenario.

```
import 'office-ui-fabric-react/dist/css/fabric.min.css';
```

### Priorities and help requested:

Requested priority: (Blocking, High, Normal, Low)
Low

Products/sites affected: (if applicable)
https://techskillsforbusiness.microsoft.com/

**Comment by marygans** [comment]
Hi @AndrewCraswell, thanks for submitting this issue! Can you provide a codepen demonstrating your issue? Thanks!

**Comment by AndrewCraswell** [comment]
@marygans Unfortunately that will be difficult since it involves a custom icon subset. Unless there is an example one already hosted online ready for such situations.

But I'd be happy to give access to our test environment with repro instructions (it's fairly straight forward).

**Comment by marygans** [comment]
Hi @AndrewCraswell its possible that this actually a dupe of #10449

**Comment by maxwellred** [comment]
Hi @AndrewCraswell I have created a local test-repo using a custom icon subset that I have investigated your issue with and can't repro the issue where ActionButton pulls in the entire icons woff versus the Icon component (It pulls in the correct custom subset woff for both). This leads me to believe that somewhere in your application you are initializing the icon font again. It would be nice to get access to your test environment and repro instructions to take a closer look if you have the time. As for the other sites that import the min-ed css file, I have been told by an authority on the subject that yes this will pull in the complete icon font as well as tons of CSS that may cause issues with your applications.
